libbeat/conditions/network.go

```diff
@@ -202,12 +211,18 @@ func parseCIDR(value string) (*net.IPNet, error) {
 
 // extractIP return an IP address if unk is an IP address string or a net.IP.
 // Otherwise it returns nil.
-func extractIP(unk interface{}) net.IP {
+func extractIP(unk interface{}) []net.IP {
 	switch v := unk.(type) {
 	case string:
-		return net.ParseIP(v)
+		return []net.IP{net.ParseIP(v)}
 	case net.IP:
```

Comment:
Not sure if/when this case comes into play, but if it expects a `net.IP`, do you think it could also encounter a `[]net.IP`?